.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
what does this line do?

Reply:
After running 'husky install', it'll add husky.sh under .husky/_/. This is to find the husky configuration file. (Seeing chat does this way.

Reply:
@alinayh Can you provide a link to the chat example?

Reply:
actually seeing this in the pages repo: https://github.com/yext/pages/blob/main/.husky/pre-commit